src/models/users.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+class UsersModel {
+  constructor() {
+    this.users = [];
+  }
+
+  reset() {
+    this.users = [];
+  }
+
+  getAll() {
+    return this.users;
+  };
+
+  getById(userId) {
+    return this.users.find(user => user.id === userId) || null;
+  }
+
+  addUser(name) {
+    const maxId = this.users.lengt
```

Comment:
```suggestion
    const maxId = this.users.length
```

So it always gets the the same for all objects. But the tests have passed :)